joomla-projects/media-manager-improvement
# Use filePathUrl instead of filePath

Pull Request for Issue #84

### Summary of Changes

* Use filePathUrl instead of filePath

### Testing Instructions

* Open media manager on a joomla instance in a subfolder.
* See image preview in the media-browser
